src/framework/preferences/js/Panels.js

```diff
@@ -546,12 +559,18 @@ var fluid_2_0 = fluid_2_0 || {};
                 var valuebinding = tree.valuebinding;
                 var modelValue = fluid.get(model, fluid.prefs.compositePanel.rebaseValueBinding(valuebinding, modelRelayRules));
                 return modelValue !== undefined ? modelValue : val;
+            } else if (key === "messagekey") {
+                // converts the "UIMessage" renderer component into a "UIBound"
+                // and passes in the resolved message as the value.
+                rebasedMessagekey.componentType = "UIBound";
```

Comment:
This should be done in place, replacing the existing structure rather than the ad-hoc scheme which assigns to an outside closure variable